### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by oofpez**

### CODE REVIEW

File: tapi/default.md [hunk 1]
```diff
@@ -424,7 +424,33 @@ An agency, or operator, is an organisation which provides and governs a transpor
...
+An alert is a message that may indicate a wide variety of effects affecting service.
+
+##### Alert response model
+
+| Field | Type | Description |
+| :--------- | :--- | :---- |
+| message | string | Text describing the alert. |
+| effect | [Alert Effect](#AlertEffect) | The effect this alert represents on the service. |
+
+##### AlertEffects
```
Comment (chrisk1ng) [inline tapi/default.md]: Should "AlertEffect"  ?

File: tapi/default.md [hunk 2]
```diff
@@ -424,7 +424,33 @@ An agency, or operator, is an organisation which provides and governs a transpor
...
+| Value | Description |
+| :--------- | :---- |
+| NoService | Service suspended. |
+| ReducedService | Service running at lowered capacity. |
+| SignificantDelays | Service running but with substantial delays expected.  |
+| Detour | Service running on alternative routes to avoid problem. |
+| AdditionalService | Service above normal capacity. |
+| ModifiedService | Service different from normal capacity. |
+| OtherEffect | Miscellaneous, undefined effect. |
+| StopMoved | Stop not at previous location or stop no longer on route. |
```
Comment (chrisk1ng) [inline tapi/default.md]: Remove StopMoved  ModifiedService  AdditionalService  Detour

File: tapi/default.md [hunk 3]
```diff
@@ -457,7 +483,14 @@ GET api/agencies?bbox=-33.93901,18.39801,-33.92101,18.44301
         "id":"CVVPBFb_v0KzC6cFAJGOkw",
         "href":"https://platform.whereismytransport.com/api/agencies/CVVPBFb_v0KzC6cFAJGOkw",
         "name":"Cape Town Taxi",
-        "culture":"en"
+        "culture":"en",
+        "alerts":
+        [
+           {
+              "message": "Heavy traffic causing significant delays of 20m on N2 Freeway.",
```
Comment (chrisk1ng) [inline tapi/default.md]: Here is an actual message: "All trains are delayed by approx 20 mins from 15:25 due to a technical problem."

File: tapi/default.md [hunk 4]
```diff
@@ -424,7 +424,30 @@ An agency, or operator, is an organisation which provides and governs a transpor
...
+##### AlertEffects
+
+Type of effect. The following table describes the alert effects currently supported by the API.
+
+| Value | Description |
+| :--------- | :---- |
+| NoService | Service suspended. |
+| ReducedService | Service running at lowered capacity. |
+| SignificantDelays | Service running but with substantial delays expected.  |
+| Detour | Service running on alternative routes to avoid problem. |
```
Comment (chrisk1ng) [inline tapi/default.md]: Remove detour please

File: tapi/default.md [hunk 5]
```diff
@@ -457,7 +480,14 @@ GET api/agencies?bbox=-33.93901,18.39801,-33.92101,18.44301
         "id":"CVVPBFb_v0KzC6cFAJGOkw",
         "href":"https://platform.whereismytransport.com/api/agencies/CVVPBFb_v0KzC6cFAJGOkw",
         "name":"Cape Town Taxi",
-        "culture":"en"
+        "culture":"en",
+        "alerts":
+        [
+           {
+              "message": "All vehicles are delayed by approx 20 mins from 15:25 due to a technical problem.",
```
Comment (chrisk1ng) [inline tapi/default.md]: All vehicles are delayed by approx 20 mins from 15:25 due to a strike.

Makes more sense for cape town taxi

Comment (oofpez) [inline tapi/default.md]: lol!

File: tapi/default.md (hunk 4 above)
Comment (oofpez) [inline tapi/default.md]: gotcha

**Comment by davenewza** [comment]
@chrisk1ng can we close this PR?

**Comment by davenewza** [comment]
Closing this.

**Pull request closed without merging**